apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/extract/subset/DataSetSubsetter.java

```diff
@@ -51,6 +47,14 @@
 import org.apache.commons.csv.CSVPrinter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.core.exception.SdkClientException;
+import software.amazon.awssdk.services.s3.model.GetObjectRequest;
+import software.amazon.awssdk.transfer.s3.S3TransferManager;
+import software.amazon.awssdk.transfer.s3.internal.DefaultS3TransferManager;
+import software.amazon.awssdk.transfer.s3.model.CompletedFileDownload;
+import software.amazon.awssdk.transfer.s3.model.DownloadFileRequest;
+import software.amazon.awssdk.transfer.s3.model.FileDownload;
+import software.amazon.awssdk.transfer.s3.progress.LoggingTransferListener;
 
 /**
  * Given some RIF data sets, extracts a random subset of beneficiaries and their claims. This is
```

Comment:
This class is definitely a candidate for /dev/nul; the CSV writer support is sketchy at best, especially with the recent change(s) in how the models are created (DSL vs. excel). Sorry you had to deal with this when it is not even used.

Reply:
Do we think this should be removed as part of this PR or as a larger effort for cleanup related to its functionality?

Reply:
> Do we think this should be removed as part of this PR or as a larger effort for cleanup related to its functionality?

This PR is on the larger side, but I think you've done well in keeping it focused. I've opened [BFD-2594](https://jira.cms.gov/browse/BFD-2594) as a placeholder to address some of this obsolete code.